google/cloud/datastore/key.py

```diff
@@ -314,6 +348,9 @@ def to_legacy_urlsafe(self, location_prefix=None):
             prefix may need to be specified to obtain identical urlsafe
             keys.
 
+        .. note::
+            to_legacy_urlsafe does not support named databases.
```

Comment:
Will that be better (or more confusing) if we call this non-default?

Reply:
Will change to instead use positive text ("only supports the default database") or something similar